content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -16,18 +16,56 @@ Since both O3DE Gems and projects use the same CMake build functions, these inst
 
 ## Build setup
 
-The first step of enabling the Multiplayer Gem for a project is to make sure that the `<ProjectName>.Static` target of your project includes the correct dependencies. Make the following edits to the `<ProjectName>.Static` target:
+As usual, the first step of enabling the Multiplayer Gem is to make sure the gem is [added to the project](/docs/user-guide/project-config/add-remove-gems/).
+After adding the Multiplayer Gem to the project there are more steps that require editing some of the project's CMake and C++ files by hand. 
+### CMakeList.txt changes
+Make sure the `<ProjectName>.Static` target includes the correct dependencies. 
+Find the CMake file defining your project's static target; for example `<ProjectName>/Code/CMakeList.txt`. 
+Make the following edits to the `<ProjectName>.Static` target:
+1. In the `FILES_CMAKE` section, add `<projectname>_autogen_files.cmake`
+    ```cmake
+        ly_add_target(
+            NAME <ProjectName>.Static STATIC
+            ...
+            FILES_CMAKE
+                ...
+                <projectname>_autogen_files.cmake
+    ```
+    {{< note >}}Creating the new `<projectname>_autogen_files.cmake` file is covered below in the [Adding AutoGen CMake file](#adding_autogen_file) section.{{< /note >}}
```

Comment:
"Create a new..",

Maybe "The <projectname>_autogen_files.cmake" file may not exist at this time. We will create this file later in the [Adding AutoGen CMake file](#adding_autogen_file) section. For now we just want to create reference to it in the CMakeList.txt"

Reply:
Fixed! Thank you